lib/player.rb

```diff
@@ -0,0 +1,8 @@
+# This a player class to save properties and capabilities of player
+
+class Player
+  attr_accessor :name
+  def initialize(name)
+    @name = name
+  end
+end
```

Comment:
## Nam and tag
- You will find easier if you add a tag property to the player object
```ruby
attr_accessor :name, :tag
  def initialize(name,tag)
    @name = name
   @tag= tag
  end
```

Reply:
Solved.